packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -70,6 +87,7 @@ export function bindCallActions(dispatch, call) {
     localMediaStreamUrl: call.localMediaStreamUrl
   };
   call.on(`connected`, () => dispatch(callConnected(call)));
```

Comment:
Good call, yup, these should probably be `.once` since we really don't want those actions to fire multiple times.